stage1/init/kvm/resources.go

```diff
@@ -68,3 +68,17 @@ func GetAppsResources(apps schema.AppList) (totalCpus, totalMem int64) {
 
 	return totalCpus, totalMem
 }
+
+// GetAppCapabilities is a filter which returns list of valid Linux capabilities
+// Function requires a manifest of application
+func GetAppCapabilities(app *types.App) []types.LinuxCapability {
```

Comment:
This function is not specific to kvm. I wonder if this can be refactored with the loop in [appToNspawnArgs](https://github.com/coreos/rkt/blob/master/stage1/init/common/pod.go#L696) somehow.